src/ai/backend/client/cli/session.py

```diff
@@ -538,7 +550,8 @@ def _create_from_template_cmd(docs: str = None):
         "-g",
         "--group",
         metavar="GROUP_NAME",
-        default=None,
```

Comment:
If a `None` value is passed, the `group` value gets overwritten with `None`, leading to an error.
>collected an error log [ERROR] "ai.backend.manager.api.exceptions.InvalidAPIParameters: Missing or invalid API parameters. (Invalid group)" from manager